core/templates/domain/question/question-backend-api.service.spec.ts

```diff
@@ -231,6 +231,10 @@ describe('Question backend Api service', () => {
       let successHandler = jasmine.createSpy('success');
       let failHandler = jasmine.createSpy('fail');
       questionBackendApiService.fetchQuestionsAsync(
+        // This throws "Type string is not assignable to type
+        // 'number'." We need to suppress this error
+        // because of the need to test validations.
+        // @ts-ignore
```

Comment:
Yes, there is some future plan of this one in my mind. Like open issue for the same. But correctly i did not make any pre plan during Milestone/proposal making. So can't do this right now as it may cause delays.

Need proper planning, issue & description.

Reply:
Yes, there is some future plan of this one in my mind. Like open issue for the same. But correctly i did not make any pre plan during Milestone/proposal making. So can't do this right now as it may cause delays.

Need proper planning, issue & description.